packages/app-store/alby/components/AlbyPriceComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import { fiat } from "@getalby/lightning-tools";
+import React from "react";
+
+import { Tooltip } from "@calcom/ui";
+import { SatSymbol } from "@calcom/ui/components/icon/SatSymbol";
+
+type AlbyPriceComponentProps = {
+  displaySymbol: boolean;
+  price: number;
+};
+
+export function AlbyPriceComponent({ displaySymbol, price }: AlbyPriceComponentProps) {
+  const [fiatValue, setFiatValue] = React.useState<string>("loading");
+  React.useEffect(() => {
+    (async () => {
+      const unformattedFiatValue = await fiat.getFiatValue({ satoshi: price, currency: "USD" });
+      setFiatValue(`$${unformattedFiatValue.toFixed(2)}`);
+    })();
+  }, [price]);
+
+  return (
+    <Tooltip content={fiatValue}>
+      <div className="inline-flex items-center justify-center">
+        {displaySymbol && <SatSymbol className="h-4 w-4" />}
+        {price}
```

Comment:
Should we add the word "sats" here?

Reply:
![image](https://github.com/calcom/cal.com/assets/4461358/1a8ce877-7a78-4f0b-b12d-2bd2f7582e34)

Reply:
@alannnc sounds good to me.

Reply:
Yes, I think that's better and more understandable for users. I've added it now. Thanks!